proselint/config_base.py

```diff
@@ -1,7 +1,26 @@
 """Proselint config - replacement for default .proselintrc since #1212."""
+from __future__ import annotations
 
-default = {
+from enum import IntEnum
+
+
+class Output(IntEnum):
+    full = default = 0
+    json = 1
+    compact = 2
+    # implement printing to file?
+
+    @classmethod
+    def names(cls) -> list[str]:
+        return [_e.name for _e in cls]
+
+
+proselint_base: dict = {
```

Comment:
Is this type not inferred? I am not very familiar with Python's type system, having worked with it briefly.

Reply:
it can have advantages for type-linting with mypy as the type is now considered static and not dynamic. specializing more would be the way to go, but the config mixes value-types